reference/3_querying_data/3_2_filters/README.md

```diff
@@ -0,0 +1,98 @@
+# Using filters
+
+## Example - Filtering an array
+
+> **_Try out:_**
+>
+> Place node **tutorials/data_qry/filter** or **[download .cranqj](cranqj/data_qry_filter.cranqj)** 
+
+> **_More info:_**
+>
+> For application flow use cases of filters, check out  **[Application flow - Iterators / loops](../../1_application_flow/1_2_iterators/README.md)**
+
+In **[Example - Concatenating arrays](../../2_constructing_data/2_3_merge_concat/README.md)**, we created an array of employee records - let's take this dataset, and implement a node, that returns all employees working for a specified department.
+
+As a reminder, here is our test dataset:
+
+### Test data:
+
+```json
+[
+  {
+    "EmpID": 101,
+    "Name": "Sue",
+    "Dept": "Facilities",
+    "HireDate": "2019-02-13",
+    "Salary": 1500
+  },
+  {
+    "EmpID": 100,
+    "Name": "Ted",
+    "Dept": "Accounting",
+    "HireDate": "2020-11-08",
+    "Salary": 1500
+  },
+  {
+    "EmpID": 1,
+    "Name": "Joan",
+    "Dept": "Accounting",
+    "HireDate": "2016-04-28",
+    "Salary": 2200
+  },
+  {
+    "EmpID": 2,
+    "Name": "Frank",
+    "Dept": "Facilities",
+    "HireDate": "2011-11-13",
+    "Salary": 2000
+  },
+  {
+    "EmpID": 3,
+    "Name": "Louis",
+    "Dept": "Legal",
+    "HireDate": "2015-09-01",
+    "Salary": 3000
+  }
+]
+```
+
+![](images/2021-07-16-17-59-48.png)
+
+- Place an instance of the node ```tutorials/data_man/concat_array``` implemented in the previous example (or create it now)
+- Then create a new node "get_by_dept" that performs the filtering
+  - Create 2 input ports, ```emps``` will take the employee records, while ```dept``` will take the deparment we are filtering against
+  - Connect the nodes as seen above
+
+![](images/2021-08-03-19-11-26.png)

```

Comment:
I'd name the store [Dept] as ["Dept"] to clearly show it holds the static value of "Dept" and it's not a node doing any processing. 
I've seen this practice in other examples (3_3_mappers) and liked it.